public/manifest.json

```diff
@@ -3,7 +3,7 @@
   "manifest_version": 3,
   "name": "Panda Wallet",
   "description": "A non-custodial and open-source wallet for BSV and 1Sat Ordinals",
-  "permissions": ["storage"],
+  "permissions": ["storage",  "unlimitedStorage"],
```

Comment:
What specifically is this for?

Reply:
The default storage limit is about 5M, it prevents transactioins which have large size to be signed by the wallet. So this permission should be granted.

Reply:
@danwag06 can you merge this